### ISSUE TITLE
pycodestyle mixed with pep8 warnings

### ISSUE THREAD

**Issue by oerp-odoo** [SublimeLinter/SublimeLinter-pycodestyle#6]
I know pep8 was renamed to pycodestyle, though sublimelinter throws some confusing warnings.
If I have `pep8` and `pycodestyle` installed, then I get this warning:

```
SublimeLinter: #2 linter.py:589: WARNING: pep8 output:
/usr/local/lib/python3.5/dist-packages/pep8.py:2124: UserWarning: 

pep8 has been renamed to pycodestyle (GitHub issue #466)
Use of the pep8 tool will be removed in a future release.
Please install and use `pycodestyle` instead.

$ pip install pycodestyle
$ pycodestyle ...

  '\n\n'
```
If I uninstall pep8, then I start getting this warning..:

```
SublimeLinter: #1 python_linter.py:101: WARNING: cannot locate 'pep8'. Fill in the 'python' or 'executable' setting.
[2018-08-11 16:25:44,106 - WARNING  - SublimeLinter.lint.base_linter.python_linter] cannot locate 'pep8'. Fill in the 'python' or 'executable' setting.
```
I tried looking into settings, but could not find anywhere where it is supposed to look for pep8. Though why it should look for pep8 if I have installed pycodestyle? I also reinstalled sublimelinter, but that did not help it.

Packages used:
```
SublimeLinter: 4.7.2
pycodestyle: 2.3.1

**Comment by braver** [comment]
You perhaps  have the SublimeLinter-pep8 plug-in installed? Can you show a full log after restarting ST?

**Comment by oerp-odoo** [comment]
@braver you are right. I had that  SublimeLinter-pep8 plug-in installed and forgot all about it. After removing it, I no longer get that warning.

**Comment by braver** [comment]
🧚🏻‍♂️
